crates/re_components/src/lib.rs

```diff
@@ -299,3 +332,25 @@ where
         v
     }
 }
+
+// ---------------------------------------------------------------------------
+
+/// Profiling macro for feature "puffin"
```

Comment:
something I noticed when copy pasting this around myself: We don't actually feature gate these and there's no feature either.